test/MHD/sphBragViscosity/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+set_idefix_property(Idefix_RECONSTRUCTION LimO3)
+enable_idefix_property(Idefix_MHD)
+replace_idefix_source(fluid/braginskii/bragViscosity.cpp noDivBragViscosity.cpp)
```

Comment:
@glesur do we have any test that uses `replace_idefix_source` already ? seems like a heavy maintenance burden

Reply:
No problem anymore, I will comment those lines and make this test a regression-check only